tpgtools/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/GoogleCloudPlatform/magic-modules/tpgtools
 
-go 1.21
+go 1.23
```

Comment:
does tpgtools/go.mod also need to be updated?

Reply:
running `go mod tidy` didn't produce any dependency changes on go.sum so no update is needed.